doc/content/modules/user-manual/pages/release-notes/2025.8.0.adoc

```diff
@@ -158,6 +158,9 @@ package p1 {
 
 - In the _Explorer_ view, fix an issue where the context menu of a tree item was loaded twice.
 
+- Fix action "New objects from text" being available on all semantic elements in the Explorer view.
+Now the backend may specify on which elements this contextual menu action is available.
```

Comment:
Explorer view => _Explorer_ view

Now the backend... => this is too technical for the release notes.

Just explain here that "New objects from text" is now only available on SysMLv2 elements.